README.md

```diff
@@ -673,10 +635,23 @@ SecurityOptions = [
 {ok, Pid} = riakc_pb_socket:start("127.0.0.1", 8087, SecurityOptions).
 ```
 
-More detailed information can be found in our [official
-documentation](http://docs.basho.com/riak/latest/dev/advanced/client-security/erlang/).
+More detailed information can be found in our [official documentation][kv_using_security_basics].
 
 Troubleshooting
 ==================
 
 If `start/2` or `start_link/2` return `{error,econnrefused}` the client could not connect to the server - make sure the protocol buffers interface is enabled on the server and the address/port is correct.
+
+[basho_docs]: http://docs.basho.com/
+[kv_setup]: http://docs.basho.com/riak/kv/latest/setup/
+[erlang_client_edocs]: http://basho.github.com/riak-erlang-client/
```

Comment:
These docs appear to be ancient, i.e., 2013.  Should we either update or remove the link?